config/mcp_config.py

```
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+"""MCP Configuration Management"""
+
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)